py/kubeflow/kfctl/testing/util/kfctl_go_test_utils.py

```diff
@@ -136,48 +137,46 @@ def filter_spartakus(spec):
       break
   return spec
 
-def get_config_spec(config_path, project, email, zone, app_path):
+def get_config_spec(config_path, app_path, cluster_name):
   """Generate KfDef spec.
 
   Args:
     config_path: Path to a YAML file containing a KFDef object.
     Can be a local path or a URI like
     https://raw.githubusercontent.com/kubeflow/manifests/master/kfdef/kfctl_gcp_iap.yaml
-    project: The GCP project to use.
-    email: a valid email of the GCP account
-    zone: a valid GCP zone for the cluster.
     app_path: The path to the Kubeflow app.
+    cluster_name: Name of EKS cluster
   Returns:
     config_spec: Updated KfDef spec
   """
   # TODO(https://github.com/kubeflow/kubeflow/issues/2831): Once kfctl
   # supports loading version from a URI we should use that so that we
   # pull the configs from the repo we checked out.
   config_spec = load_config(config_path)
-  apiVersion = config_spec["apiVersion"].strip().split("/")
-  if len(apiVersion) != 2:
-    raise RuntimeError("Invalid apiVersion: " + config_spec["apiVersion"].strip())
-  if apiVersion[-1] == "v1alpha1":
-    config_spec["spec"]["project"] = project
-    config_spec["spec"]["email"] = email
-    config_spec["spec"]["zone"] = zone
-  elif apiVersion[-1] in ["v1beta1", "v1"]:
-    for plugin in config_spec["spec"].get("plugins", []):
-      if plugin.get("kind", "") == "KfGcpPlugin":
-        plugin["spec"]["project"] = project
-        plugin["spec"]["email"] = email
-        plugin["spec"]["zone"] = zone
-        break
-  else:
-    raise RuntimeError("Unknown version: " + apiVersion[-1])
-  config_spec["spec"] = filter_spartakus(config_spec["spec"])
+  # apiVersion = config_spec["apiVersion"].strip().split("/")
```

Comment:
This is GCP specific configs. Please remove it if you don't need them

Reply:
Removed